Cargo.lock

```diff
@@ -1,3 +1,5 @@
+# This file is automatically @generated by Cargo.
```

Comment:
Is this comment autogenerated?

Reply:
Yes, it's autogenerated by `cargo fix --edition`